### DISCUSSION TITLE
Display the coordinates of pixels of the same color

### DISCUSSION THREAD

**Discussion by b1zar** [ImageMagick/ImageMagick#2506] [Help] [answered]
Hey. Tell me how to display the coordinates of pixels of the same color with the ability to specify the color encoded rgb (0,0,0) or #FFFFFFF.

Windows 10
ImageMagick-7.0.10-24-Q16-HDRI-x64-dll

**Comment by snibgo** [comment, marked as answer]
Make all the other colours transparent, and write to the pseudo-format "sparse-color:". Like this:
```
magick in.png +transparent sRGB(10%,20%,30%) sparse-color:
```

**Reply by b1zar** [reply]
Thank. Tried this command before asking, but the output was empty, apparently there was an error somewhere. And how to make the coordinates be line by line, and not all in one line?

**Comment by snibgo** [comment]
> ...  but the output was empty, ...

I suppose the image had no pixels with the exact colour you specified. If you think otherwise, then please post your command and the input image. You can use "-fuzz" for approximate matches.

> And how to make the coordinates be line by line, and not all in one line?

Pipe the text output through a tool that converts newlines to spaces, eg:
```
magick in.png +transparent sRGB(10%,20%,30%) sparse-color: | tr " " "\n"
```
